_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -147,5 +147,7 @@ To set up an alert for a detector, continue with the following steps:
 ---
 ## What's next
 
-If you are ready to view findings for the new detector, see the [Working with findings]({{site.url}}{{site.baseurl}}/security-analytics/usage/findings/) section. If you would like to import rules or set up custom rules before working with findings, see the [Working with rules]({{site.url}}{{site.baseurl}}/security-analytics/usage/rules/) section. 
+If you are ready to view findings for the new detector, see the [Working with findings]({{site.url}}{{site.baseurl}}/security-analytics/usage/findings/) section. If you would like to import rules or set up custom rules before working with findings, see the [Working with detection rules]({{site.url}}{{site.baseurl}}/security-analytics/usage/rules/) section.
```

Comment:
Is "for" the right preposition after "findings" here? The meaning of the first phrase is slightly ambiguous.

Reply:
changed to "generated by".